Core/Inc/MQ131.h

```diff
@@ -0,0 +1,5 @@
+#ifndef MQ131_H
+#define MQ131_H
+uint32_t Read_ADC(void);
+float Voltage_to_PPB(float voltage);

```

Comment:
Fix variable names. Functions that start with a capital are classes (as specified in ECE 150 and mostly everywhere else)